public/controllers/agent/register-agent/utils.ts

```diff
@@ -0,0 +1,51 @@
+import { WzRequest } from '../../../react-services';
+import { ServerAddressOptions } from './steps/server-address';
+
+/**
+ * Get a list of nodes and parsed into a string separated by semicolon
+ * @param selectedNodes
+ */
+export const parseNodeIPs = (selectedNodes: any): string => {
+  let allNodeIps = '';
+  if (selectedNodes.length > 1) {
+    allNodeIps = selectedNodes.map((o) => o.value).join(';');
```

Comment:
**nipick**: According to the documentation, the separator character of the manager IPs is `,`.

Seen in the examples of:
Linux: https://documentation.wazuh.com/current/user-manual/deployment-variables/deployment-variables-linux.html#deployment-variables-for-linux
macOs: https://documentation.wazuh.com/current/user-manual/deployment-variables/deployment-variables-macos.html
AIX: https://documentation.wazuh.com/current/user-manual/deployment-variables/deployment-variables-aix.html

I don't know about Windows, we could ask.

Reply:
That's right.
As mentioned @gdiazlo, in the linked [issue](https://github.com/wazuh/wazuh-kibana-app/issues/4109#issuecomment-1285995336), in windows the separator character is different `;`
[https://documentation.wazuh.com/current/user-manual/deployment-variables/deployment-variables-windows.html](https://documentation.wazuh.com/current/user-manual/deployment-variables/deployment-variables-windows.html),
I will add this condition to the code.

Reply:
Solved